Doc/library/getopt.rst

```diff
@@ -162,8 +169,15 @@ and more informative help and error messages by using the :mod:`argparse` module
        # ... do something with args.output ...
        # ... do something with args.verbose ..
 
+In more complex cases (such as options which accept values), the behaviour
+of the ``argparse`` version may diverge from that of the ``getopt`` and
+``optparse`` versions due to the way ``argparse`` handles parameter
+values that start with ``-``.
```

Comment:
The difference is not only in this.

Reply:
Now covered at the start of the optparse docs.